R/cal-plot-regression.R

```diff
@@ -95,6 +91,10 @@ regression_plot_impl <- function(.data, truth, estimate, group,
   estimate <- enquo(estimate)
   group <- enquo(group)
 
+  if (quo_is_null(group)) {
+    .data[[".config"]] <- NULL
+  }
```

Comment:
If there isn't a group variable, e.i. `.config` is constant, it is removed from the plotting data.

This makes it easier to test that faceting isn't done with constant `.config`